airbyte-integrations/bases/base-normalization/normalization/transform_catalog/transform.py

```diff
@@ -162,29 +159,36 @@ def find_combining_schema(properties: dict) -> set:
     return set(properties).intersection({"anyOf", "oneOf", "allOf"})
 
 
+def jinja_call(command: str) -> str:
+    return "{{ " + command + "  }}"
+
+
 def json_extract_base_property(path: List[str], json_col: str, name: str, definition: dict) -> Optional[str]:
     current = path + [name]
     if "type" not in definition:
         return None
     elif is_string(definition["type"]):
-        return (
-            f"cast({MACRO_START} json_extract_scalar('{json_col}', {current}) {MACRO_END} as {MACRO_START} dbt_utils.type_string()"
-            + f" {MACRO_END}) as {MACRO_START} adapter.quote_as_configured('{name}', 'identifier') {MACRO_END}"
+        return "cast({} as {}) as {}".format(
+            jinja_call(f"json_extract('{json_col}', {current})"),
+            jinja_call("dbt_utils.type_string()"),
+            jinja_call(f"adapter.quote_as_configured('{name}', 'identifier')"),
         )
     elif is_integer(definition["type"]):
-        return (
-            f"cast({MACRO_START} json_extract_scalar('{json_col}', {current}) {MACRO_END} as {MACRO_START} dbt_utils.type_int()"
-            + f" {MACRO_END}) as {MACRO_START} adapter.quote_as_configured('{name}', 'identifier') {MACRO_END}"
+        return "cast({} as {}) as {}".format(
+            jinja_call(f"json_extract_scalar('{json_col}', {current})"),
```

Comment:
nice. so we didn't end up having to do anything integration specific in the dbt stuff. we were able to use the jinja macros!

Reply:
Yes, I had to write ugly DBT SQL (with macro everywhere) so it's the most generic possible and we handle destination specific in the DBT macros